docker/api/daemon.py

```diff
@@ -67,11 +84,13 @@ def events(self, since=None, until=None, filters=None, decode=None):
             'filters': filters
         }
         url = self._url('/events')
+        response = self._get(url, params=params, stream=True, timeout=None)
+        stream = self._stream_helper(response, decode=decode)
 
-        return self._stream_helper(
-            self._get(url, params=params, stream=True, timeout=None),
-            decode=decode
-        )
+        if cancellable:
```

Comment:
I think we can get rid of this, and simply always return a `CancellableStream`, provided it behaves the same as the regular stream.